### ISSUE TITLE
Attachment URL

### ISSUE THREAD

**Issue by devamchakrabarty** [OfficeDev/office-js-docs-reference#705]
For `cloud` type of attachments, [this page](https://docs.microsoft.com/en-us/javascript/api/outlook/office.mailboxenums.attachmenttype?view=outlook-js-preview) states 
> The id property of the attachment contains a URL to the file.

is this information version specific? 

[Another page](https://docs.microsoft.com/en-us/javascript/api/outlook/office.attachmentdetails?view=outlook-js-preview) gives us a separate 
property called `url` for this purpose (Mailbox v1.8).

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 0ba54eb2-3b60-c005-a690-b7e08669536f
* Version Independent ID: d5d3255e-668a-7247-f059-c95951b7506c
* Content: [Office.MailboxEnums.AttachmentType enum - Office Add-ins](https://docs.microsoft.com/en-us/javascript/api/outlook/office.mailboxenums.attachmenttype?view=outlook-js-preview)
* Content Source: [docs/docs-ref-autogen/outlook/outlook/office.mailboxenums.attachmenttype.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/live/docs/docs-ref-autogen/outlook/outlook/office.mailboxenums.attachmenttype.yml)
* Product: **outlook**
* Technology: **add-ins**
* GitHub Login: @o365devx
* Microsoft Alias: **o365devx**

**Comment by AlexJerabek** [comment]
Thank you for the question @devamchakrabarty. @ElizabethSamuel-MSFT, could you please investigate the answer?

**Comment by ElizabethSamuel-MSFT** [comment]
@devamchakrabarty Thanks for pointing this out. The cloud item description should reference the url property instead of the id property. We'll get this updated.

Thanks.

**Comment by ElizabethSamuel-MSFT** [comment]
@devamchakrabarty The description for the `cloud` value has been updated to indicate where the URL can be found based on Compose vs Read mode. We've also taken the opportunity to define separate objects to represent attachment details in Read (Office.AttachmentDetails) vs Compose (Office.AttachmentDetailsCompose).

We've updated the docs and the changes should be published soon.

Let us know if your need further clarification or face any other issues.

Thanks.
